lib/fastlane/plugin/tunes/actions/tunes_action.rb

```diff
@@ -1,25 +1,56 @@
+require 'shellwords'
 module Fastlane
   module Actions
     class TunesAction < Action
       def self.run(params)
-        sh "afplay '#{params[:file_path]}'"
+        if params[:file_path]
+          play_song(params[:file_path])
+        elsif params[:directory]
+          UI.message("Playing all songs from directory '#{params[:directory]}'")
+          Dir[File.join(params[:directory], "*.mp3")].each do |current|
+            play_song(current)
+          end
+        else
+          UI.user_error!("Please provide either a file_path or a directory")
+        end
+      end
+
+      def self.play_song(path)
+        Helper::TunesHelper.print_audio_information(path)
+        sh "afplay #{path.shellescape}"
       end
 
       def self.description
-        "Play music using fastlane, because you can."
+        "Play music using fastlane, because you can"
```

Comment:
Nope, no dots allowed: https://github.com/fastlane/fastlane/blob/master/fastlane/spec/actions_list_spec.rb#L33
I shall add a linter for that